src/PlacesAutocomplete.js

```diff
@@ -153,7 +153,7 @@ class PlacesAutocomplete extends React.Component {
       this.clearAutocomplete()
       return;
     }
-    this.autocompleteService.getPlacePredictions({ input: event.target.value }, this.autocompleteCallback)
+    this.autocompleteService.getPlacePredictions({ ...{ input: event.target.value }, ...this.props.options }, this.autocompleteCallback)
```

Comment:
@cburbank Shouldn't this be 

```javascript
{ ...this.props.options, input: event.target.value }
```

to make sure that `input: event.target.value` has precedence over the option object?